src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -68,9 +71,14 @@ export class HuiDialogEditCard extends LitElement {
         `ui.panel.lovelace.editor.card.${this._cardConfig.type}.name`
       )} ${this.hass!.localize("ui.panel.lovelace.editor.edit_card.header")}`;
     } else {
```

Comment:
this should be a else if:
```
} else if (!this._cardConfig) {
    heading = this._viewConfig.title
          ? this.hass!.localize(
              "ui.panel.lovelace.editor.edit_card.pick_card_view_title",
              "name",
              '"' + this._viewConfig.title + '"'
            )
          : this.hass!.localize("ui.panel.lovelace.editor.edit_card.pick_card");
} else {
    heading = this.hass!.localize(
        "ui.panel.lovelace.editor.edit_card.header"
    );
}

Reply:
Got it know, thanks 👍 Will change it as soon as I am home.

Reply:
Fixed with https://github.com/home-assistant/home-assistant-polymer/pull/4083/commits/53cf0a92aac3b9431219ba0c485b307b16746f5a :)